client/src/components/Toplist/Toplist.js

```diff
@@ -42,8 +42,8 @@ const Toplist = () => {
       <h2>Toplist</h2>
       {movies.map((movie, index) => {
         return (
-          <div>
-            <Ranking>#{index + 1}</Ranking>
+          <div key={index}>
+            <Ranking key={index}>#{index + 1}</Ranking>
```

Comment:
Only the other one needs a `key` and it should never be the `index`. Use the `_id` instead!